query/src/storages/information_schema/columns_table.rs

```diff
@@ -50,30 +50,7 @@ impl ColumnsTable {
             NULL AS collation_name,
             NULL AS domain_catalog,
             NULL AS domain_schema,
-            NULL AS domain_name,
```

Comment:
The removed lines is not used anymore?

Reply:
In planner v2 use lower case.

So if the table has same column name will return error.

```SQL
'root'@mysqldb 19:00:53 [(none)]> SET enable_planner_v2=1;
Query OK, 0 rows affected (0.06 sec)

'root'@mysqldb 19:00:54 [(none)]> create table tt(id int, ID int);
Query OK, 0 rows affected (0.07 sec)

'root'@mysqldb 19:00:57 [(none)]> select * from tt where id=1;
ERROR 1105 (HY000): Code: 1065, displayText = error: 
  --> SQL:1:24
  |
1 | select * from tt where id=1
  |                        ^^ column reference is ambiguous


```

So that the upper case column in the view is no sense.